src/queries.ts

```
@@ -118,6 +118,7 @@ export const runQueries = async <T>(
           query && path ? `${path.replace(/queries\[\d+\]\./, '')} = ${JSON.stringify(instruction)}` : null,
         path: path,
         details,
+        code: result.error.code,
```

Just as a sanity check cause I have not checked myself: Will the response `error` object always return an error code?
What happens if one doesn't exist?

---

Good thinking! Should we return the error code if it exists, or else return something like `INVAILD_QUERY_ERROR`?

---

I think for now we should failover return `undefined` or `null`.
IMO if the `handler` service doesn't give us ones then it's better to keep it empty otherwise we could get conflicting errors when debugging issues.

---

Like this `code: result.error.code || null`?